interfaces/acados_template/setup.py

```diff
@@ -35,13 +35,13 @@
 
 import sys
 print(sys.version_info)
+
 if sys.version_info < (3,5):
     sys.exit('Python version 3.5 or later required. Exiting.')
 
-
 setup(name='acados_template',
    version='0.1',
-   python_requires='>=3.5',
+   python_requires='>=3.5, <3.9',
```

Comment:
Some users have had problems with Python 3.8. Has this been tested?

Reply:
Ok, I think this was done by Tommy.
I will put <3.8 instead